src/core/client/admin/routes/Moderate/ModerateSearchBar/SeeAllOption.tsx

```diff
@@ -27,10 +27,10 @@ const SeeAllOption: FunctionComponent<Props> = ({
       {...rest}
     >
       <a className={styles.link} href={href || "#"} tabIndex={-1}>
-        <Localized id="moderate-searchBar-seeAllResults">
+        <Localized id="moderate-searchBar-seeAllResults" size="md">
```

Comment:
I'm guessing `size="md"` was supposed to be on the `Icon` not the `Localized`?

Reply:
Yeah, supposed to be on the icon, I think I mis-typed that.